### PR TITLE
Milestone 7: Mobile menu pull request

### PR DESCRIPTION
**Pull request by HussenYesufAlli**
In this milestone, we:

- Use Js code to add the mobile menu

### CODE REVIEW

**Review by AlphaNtihinduka — COMMENTED** [review]
Hello Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

- This is a pair programming activity. And according to the project requirements there must be the contributions from your partner. Kindly reach out to your partner to contribute to the project milestone.
- In addition, make sure that the files changes for the mobile menu are available for review ☺️.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't contain commits from you @HussenYesufAlli 

- This is a pair programming activity. And according to the project requirements, there must be contributions from you. Kindly add some contributions to this Pull Request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -85,14 +85,15 @@ main {
...
+  padding-right: 30px;
   color: blue;
 }
 
 .btn-humburger {
   border: white;
+  background-color: white;
 }
 
 /* The headline section bg-shapes */
```
Comment (usorfaitheloho) [inline style.css:103]: You did well in implementing the hamburger menu feature, please kindly increase the width of the hamburger menu so it looks similar to the Figma design given and gives you a better UI 😄 

Figma design :
image

Your design :
image

File: style.css [hunk 2]
```diff
@@ -69,12 +69,12 @@ main {
...
 .header-mobile {
-  padding: 20px;
+  padding: 40px;
   display: flex;
   width: 90%;
   align-items: center;
   justify-content: end;
-  padding-right: 10px;
+  padding-right: 0;
 }
```
Comment (usorfaitheloho) [inline style.css]: Kindly reduce the padding between the mobile menu nav items and the padding-top too, so it'd give you a better user experience and replicates the Figma design 😄 

Figma design :
image

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @HussenYesufAlli and @ntimanu ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @HussenYesufAlli  , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**